### PR TITLE
298 feature request add latex to existing formulas from chapter 8 en 1992 1 1

### LINKED ISSUE #298: [✨ Feature request]: Add LaTeX to existing formulas from chapter 8 EN 1992-1-1
### Description

afbeelding

Add LaTeX to all the above

### Acceptance Criteria

- [X] Add LateX and write tests

### Dependencies

_No response_

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by GerjanDorgelo**
## Description

Adds all LaTeX code to the existing formula from chapter 8 from 1992-1-1. Changed the diameter-sign to a \Phi to make it Default LaTeX compatible.

Fixes #298

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_18.py [hunk 1]
```diff
@@ -46,7 +46,37 @@ def test_integration_with_form_8_dot_16(self) -> None:
...
         manually_calculated_result = 168  # mm
 
-        assert form_8_17 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+        assert form_8_18 == pytest.approx(expected=manually_calculated_result, rel=1e-4)
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            ("complete", r"l_{pt1} = 1.2 \cdot l_{pt} = 1.2 \cdot 140.00 = 168.00"),
+            ("short", r"l_{pt1} = 168.00"),
```
Comment (egarciamendez) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_18.py:58]: should be lpt2
afbeelding
```suggestion
            ("complete", r"l_{pt2} = 1.2 \cdot l_{pt} = 1.2 \cdot 140.00 = 168.00"),
            ("short", r"l_{pt2} = 168.00"),
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_18.py [hunk 2]
```diff
@@ -35,3 +36,13 @@ def _evaluate(
         """Evaluates the formula, for more information see the __init__ method."""
         raise_if_negative(l_pt=l_pt)
         return 1.2 * l_pt
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.18."""
+        return LatexFormula(
+            return_symbol=r"l_{pt1}",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_18.py:43]: afbeelding
```suggestion
            return_symbol=r"l_{pt2}",
```

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_16.py [hunk 3]
```diff
@@ -127,3 +127,38 @@ def test_raise_error_if_zero_f_bpt(self) -> None:
...
             )
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            (
+                "complete",
+                (
+                    r"l_{pt} = \alpha_1 \cdot \alpha_2 \cdot \Phi \cdot \frac{\sigma_{pm0}}{f_{bpt}} = "
+                    r"1.00 \cdot 0.25 \cdot 8.00 \cdot \frac{350.00}{5.00} = 140.00"
```
Comment (egarciamendez) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_16.py:138]: afbeelding
```suggestion
                    r"l_{pt} = \alpha_1 \cdot \alpha_2 \cdot Ø \cdot \frac{\sigma_{pm0}}{f_{bpt}} = "
                    r"1.00 \cdot 0.25 \cdot 8.00 \cdot \frac{350.00}{5.00} = 140.00"
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py [hunk 4]
```diff
@@ -76,6 +77,18 @@ def _evaluate(
         raise_if_less_or_equal_to_zero(f_bpt=f_bpt)
         return alpha_1 * alpha_2 * diameter * sigma_pm0 / f_bpt
 
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.16."""
+        return LatexFormula(
+            return_symbol=r"l_{pt}",
+            result=f"{self:.2f}",
+            equation=r"\alpha_1 \cdot \alpha_2 \cdot \Phi \cdot \frac{\sigma_{pm0}}{f_{bpt}}",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py:85]:
```suggestion
            equation=r"\alpha_1 \cdot \alpha_2 \cdot Ø \cdot \frac{\sigma_{pm0}}{f_{bpt}}",
```

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_14.py [hunk 5]
```diff
@@ -52,3 +52,25 @@ def test_raise_error_if_n_b_is_negative(self) -> None:
 
         with pytest.raises(NegativeValueError):
             Form8Dot14EquivalentDiameterBundledBars(diameter=diameter, n_b=n_b)
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            (
+                "complete",
+                (r"\Phi_n = \min \left(55, \Phi \cdot \sqrt{n_b} \right) = \min \left(55, 16.00 \cdot \sqrt{4.00} \right) = 32.00"),
```
Comment (egarciamendez) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_14.py:61]: afbeelding
```suggestion
                (r"Ø_n = \min \left(55 \ \text{mm}, Ø \cdot \sqrt{n_b} \right) = \min \left(55 \ \text{mm}, 16.00 \cdot \sqrt{4.00} \right) = 32.00"),
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_14.py [hunk 6]
```diff
@@ -46,3 +47,13 @@ def _evaluate(diameter: MM, n_b: DIMENSIONLESS) -> MM:
...
         return min(diameter * np.sqrt(n_b), 55)
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.14."""
+        return LatexFormula(
+            return_symbol=r"\Phi_n",
+            result=f"{self:.2f}",
+            equation=r"\min \left(55, \Phi \cdot \sqrt{n_b} \right)",
+            numeric_equation=rf"\min \left(55, {self.diameter:.2f} \cdot \sqrt{{{self.n_b:.2f}}} \right)",
+            comparison_operator_label="=",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_14.py:58]:
```suggestion
            return_symbol=r"Ø_n",
            result=f"{self:.2f}",
            equation=r"\min \left(55 \ \text{mm}, Ø \cdot \sqrt{n_b} \right)",
            numeric_equation=rf"\min \left(55 \ \text{mm}, {self.diameter:.2f} \cdot \sqrt{{{self.n_b:.2f}}} \right)",
            comparison_operator_label="=",
```

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_11.py [hunk 7]
```diff
@@ -136,10 +136,10 @@ def test_integration_with_form_8_3(self) -> None:
         [
             (
                 "complete",
-                r"l_{0,min} = \max \left\{0.3 \cdot \alpha_6 \cdot l_{b,rqd}; 15 \cdot Ø; 200 \ \text{mm}\right\} = \max \left\{0.3 \cdot 1.00 "
+                r"l_{\Phi,min} = \max \left\{0.3 \cdot \alpha_6 \cdot l_{b,rqd}; 15 \cdot \Phi; 200 \ \text{mm}\right\} = \max \left\{0.3 \cdot 1.00 "
```
Comment (egarciamendez) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_11.py:139]:
```suggestion
                r"l_{Ø,min} = \max \left\{0.3 \cdot \alpha_6 \cdot l_{b,rqd}; 15 \cdot Ø; 200 \ \text{mm}\right\} = \max \left\{0.3 \cdot 1.00 "
```

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_11.py [hunk 8]
```diff
@@ -136,10 +136,10 @@ def test_integration_with_form_8_3(self) -> None:
...
                 r"\cdot 450.00; 15 \cdot 8; 200\right\} = 200.00",
             ),
-            ("short", "l_{0,min} = 200.00"),
+            ("short", r"l_{\Phi,min} = 200.00"),
```
Comment (egarciamendez) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_11.py:142]:
```suggestion
            ("short", r"l_{Ø,min} = 200.00"),
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_11.py [hunk 9]
```diff
@@ -64,15 +64,15 @@ def _evaluate(
     def latex(self) -> LatexFormula:
         """Returns a representation of the formula in LaTeX format."""
         arg_1_equation = r"0.3 \cdot \alpha_6 \cdot l_{b,rqd}"
-        arg_2_equation = r"15 \cdot Ø"
+        arg_2_equation = r"15 \cdot \Phi"
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_11.py:67]:
```suggestion
        arg_2_equation = r"15 \cdot Ø"
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_11.py [hunk 10]
```diff
@@ -64,15 +64,15 @@ def _evaluate(
...
         arg_3_equation = r"200 \ \text{mm}"
 
         arg_1_numerical_equation = rf"0.3 \cdot {self.alpha_6:.2f} \cdot {self.l_b_rqd:.2f}"
         arg_2_numerical_equation = rf"15 \cdot {self.diameter}"
         arg_3_numerical_equation = r"200"
 
         return LatexFormula(
-            return_symbol=r"l_{0,min}",
+            return_symbol=r"l_{\Phi,min}",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_11.py:75]:
```suggestion
            return_symbol=r"l_{Ø,min}",
```

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_1.py [hunk 11]
```diff
@@ -92,3 +92,36 @@ def test_raise_error_when_negative_f_cd_is_given(self) -> None:
...
             )
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            (
+                "complete",
+                (
+                    r"\Phi_{m,min} = \frac{F_{bt} \left( \frac{1}{a_b} + \frac{1}{2 \cdot \Phi} \right) }{f_{cd}} "
+                    r"= \frac{80.00 \cdot 1000 \cdot \left( \frac{1}{200.00} + \frac{1}{2 \cdot 16.00} \right)}{30.00} = 96.67"
```
Comment (egarciamendez) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_formula_8_1.py:103]: afbeelding
```suggestion
                    r"Ø_{m,min} = \frac{F_{bt} \left( \frac{1}{a_b} + \frac{1}{2 \cdot Ø} \right) }{f_{cd}} "
                    r"= \frac{80.00 \cdot 1000 \cdot \left( \frac{1}{200.00} + \frac{1}{2 \cdot 16.00} \right)}{30.00} = 96.67"
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_1.py [hunk 12]
```diff
@@ -61,3 +62,16 @@ def _evaluate(
             f_cd=f_cd,
         )
         return f_bt * KN_TO_N * ((1 / a_b) + 1 / (2 * diameter)) / f_cd
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.1."""
+        return LatexFormula(
+            return_symbol=r"\Phi_{m,min}",
+            result=f"{self:.2f}",
+            equation=r"\frac{F_{bt} \left( \frac{1}{a_b} + \frac{1}{2 \cdot \Phi} \right) }{f_{cd}}",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_1.py:71]:
```suggestion
            return_symbol=r"Ø_{m,min}",
            result=f"{self:.2f}",
            equation=r"\frac{F_{bt} \left( \frac{1}{a_b} + \frac{1}{2 \cdot Ø} \right) }{f_{cd}}",
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_9.py [hunk 13]
```diff
@@ -69,3 +70,16 @@ def _evaluate(
         )
         raise_if_less_or_equal_to_zero(diameter_l=diameter_l)
         return min(f_wd, N_TO_KN * 16 * a_s * f_cd * (diameter_t / diameter_l))
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.9."""
+        return LatexFormula(
+            return_symbol=r"F_{btd}",
+            result=f"{self:.2f}",
+            equation=r"\min \left( F_{wd}, 16 \cdot A_s \cdot f_{cd} \cdot \frac{\Phi_t}{\Phi_l} \right)",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_9.py:79]:
```suggestion
            equation=r"\min \left( F_{wd}, 16 \cdot A_s \cdot f_{cd} \cdot \frac{Ø_t}{Ø_l} \right)",
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py [hunk 14]
```diff
@@ -72,6 +73,16 @@ def _evaluate(
         )
         return min(l_td * diameter_t * sigma_td * N_TO_KN, f_wd)
 
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.8N."""
+        return LatexFormula(
+            return_symbol=r"F_{btd}",
+            result=f"{self:.2f}",
+            equation=r"\min\left( l_{td} \cdot \Phi_t \cdot \sigma_{td}, F_{wd} \right)",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py:81]:
```suggestion
            equation=r"\min\left( l_{td} \cdot Ø_t \cdot \sigma_{td}, F_{wd} \right)",
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py [hunk 15]
```diff
@@ -127,6 +138,18 @@ def _evaluate(
         raise_if_less_or_equal_to_zero(sigma_td=sigma_td)
         return min(1.16 * diameter_t * (f_yd / sigma_td) ** 0.5, l_t)
 
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.8N transverse bar."""
+        return LatexFormula(
+            return_symbol=r"F_{wd}",
+            result=f"{self:.2f}",
+            equation=r"\min\left(l_t, 1.16 \cdot Phi_t \cdot \sqrt{\frac{f_{yd}}{\sigma_{td}}} \right)",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py:146]:
```suggestion
            equation=r"\min\left(l_t, 1.16 \cdot Ø_t \cdot ({\frac{f_{yd}}{\sigma_{td}}})^{0.5} \right)",
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py [hunk 16]
```diff
@@ -127,6 +138,18 @@ def _evaluate(
         raise_if_less_or_equal_to_zero(sigma_td=sigma_td)
         return min(1.16 * diameter_t * (f_yd / sigma_td) ** 0.5, l_t)
 
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 8.8N transverse bar."""
+        return LatexFormula(
+            return_symbol=r"F_{wd}",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py:144]:
```suggestion
            return_symbol=r"l_{td}",
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py [hunk 17]
```diff
@@ -127,6 +138,18 @@ def _evaluate(
...
+            result=f"{self:.2f}",
+            equation=r"\min\left(l_t, 1.16 \cdot Phi_t \cdot \sqrt{\frac{f_{yd}}{\sigma_{td}}} \right)",
+            numeric_equation=(
+                rf"\min\left({self.l_t:.2f}, 1.16 \cdot {self.diameter_t:.2f} \cdot \sqrt{{\frac{{{self.f_yd:.2f}}}{{{self.sigma_td:.2f}}}}} \right)"
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_8n.py:148]:
```suggestion
                rf"\min\left({self.l_t:.2f}, 1.16 \cdot {self.diameter_t:.2f} \cdot ({{\frac{{{self.f_yd:.2f}}}{{{self.sigma_td:.2f}}}}})^{{0.5}} \right)"
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_7.py [hunk 18]
```diff
@@ -48,7 +48,7 @@ def latex(self) -> LatexFormula:
         return LatexFormula(
             return_symbol=r"l_{b,min}",
             result=f"{self:.2f}",
-            equation=latex_max_curly_brackets(r"0.6 \cdot l_{b,rqd}", r"10 \cdot Ø", r"100 \ \text{mm}"),
+            equation=latex_max_curly_brackets(r"0.6 \cdot l_{b,rqd}", r"10 \cdot \Phi", r"100 \ \text{mm}"),
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_7.py:51]:
```suggestion
            equation=latex_max_curly_brackets(r"0.6 \cdot l_{b,rqd}", r"10 \cdot Ø", r"100 \ \text{mm}"),
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_6.py [hunk 19]
```diff
@@ -48,7 +48,7 @@ def latex(self) -> LatexFormula:
         return LatexFormula(
             return_symbol=r"l_{b,min}",
             result=f"{self:.2f}",
-            equation=latex_max_curly_brackets(r"0.3 \cdot l_{b,rqd}", r"10 \cdot Ø", r"100 \ \text{mm}"),
+            equation=latex_max_curly_brackets(r"0.3 \cdot l_{b,rqd}", r"10 \cdot \Phi", r"100 \ \text{mm}"),
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_6.py:51]:
```suggestion
            equation=latex_max_curly_brackets(r"0.3 \cdot l_{b,rqd}", r"10 \cdot Ø", r"100 \ \text{mm}"),
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_3.py [hunk 20]
```diff
@@ -52,7 +52,7 @@ def _evaluate(
 
     def latex(self) -> LatexFormula:
         """Returns a LatexFormula object for this formula."""
-        latex_diameter = r"Ø"
+        latex_diameter = r"\Phi"
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_3.py:55]:
```suggestion
        latex_diameter = r"Ø"
```

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_2.py [hunk 21]
```diff
@@ -135,7 +135,7 @@ def latex(self) -> LatexFormula:
         return LatexFormula(
             return_symbol=r"\eta_2",
             result=f"{self:.2f}",
-            equation=r"\begin{matrix} 1.0 & \text{for }Ø ≤ 32 \\ (132 - Ø) / 100 & \text{for }Ø > 32  \end{matrix}",
+            equation=r"\begin{matrix} 1.0 & \text{for }\Phi ≤ 32 \\ (132 - \Phi) / 100 & \text{for }\Phi > 32  \end{matrix}",
```
Comment (egarciamendez) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_2.py:138]:
```suggestion
            equation=r"\begin{matrix} 1.0 & \text{for }Ø ≤ 32 \\ (132 - Ø) / 100 & \text{for }Ø > 32  \end{matrix}",
```

**Review by egarciamendez — CHANGES_REQUESTED** [review]
@GerjanDorgelo we have some issues when displaying \Phi so we have to go back to Ø. Also I found some small errors in the latex method of a couple of formulas.

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_2.py (hunk 21 above)
Comment (GerjanDorgelo) [reply blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_2.py:138]: Thank you for the feedback. the \Phi is indeed not perfect, lets change it back! Ill resolve the other issues later this week.

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_sub_formula_8_8n_3.py [hunk 22]
```diff
@@ -44,3 +44,24 @@ def test_integration_with_sub_form_8_8n_function_x(self) -> None:
         manually_result = 0.045314993
 
         assert sub_form_8_8n_3 == pytest.approx(expected=manually_result, rel=1e-4)
+
+    @pytest.mark.parametrize(
+        ("representation", "expected"),
+        [
+            ("complete", r"y = 0.015 + 0.14 \cdot e^{-0.18 \cdot x} = 0.015 + 0.14 \cdot e^{-0.18 \cdot 8.50} = 0.045"),
```
[the remark this answers is not in the document]
Comment (GerjanDorgelo) [reply tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/test_sub_formula_8_8n_3.py:51]: :)

**Review by egarciamendez — APPROVED** [review]

**Pull request merged**